modules/installation-launching-installer.adoc

```diff
@@ -54,6 +55,9 @@ ifeval::["{context}" == "installing-alibaba-network-customizations"]
 :custom-config:
 :single-step:
 endif::[]
+ifeval::["{context}" == "installing-alibaba-vpc"]
+:custom-config:
+endif::[]
```

Comment:
```suggestion
ifeval::["{context}" == "installing-alibaba-vpc"]
:custom-config:
:single-step:
endif::[]
```
Please add `:single-step:` here and `!:single-step:` after line 557. This should correct the formatting bug in the "deploying the cluster" section:

![Screenshot 2023-02-10 at 3 32 47 PM](https://user-images.githubusercontent.com/93731574/218192068-b0c7991e-9eeb-4a2c-a781-3e3e0e7e4428.png)

Reply:
@bscott-rh This is done, thanks so much for this tip!